stories/AxisCollection.stories.js

```diff
@@ -99,37 +99,37 @@ export const events = () => (
         onMouseLeave={action('onMouseLeave')}
       />
     </DataProvider>
-  </React.Fragment>
+  </>
 );
 
 events.story = {
   name: 'events',
 };
 
 export const placement = () => (
-  <React.Fragment>
+  <>
     <DataProvider defaultLoader={staticLoader} timeDomain={staticXDomain}>
       <Series id="1" color="steelblue" />
       <Series id="2" color="maroon" />
       <AxisCollection height={300} yAxisPlacement={AxisPlacement.LEFT} />
       <AxisCollection height={300} yAxisPlacement={AxisPlacement.RIGHT} />
     </DataProvider>
-  </React.Fragment>
+  </>
 );
 
 placement.story = {
   name: 'placement',
 };
 
 export const yAxisWidthStory = () => (
-  <React.Fragment>
+  <>
```

Comment:
We don't need these as fragments, right?

Reply:
Nope we don't. Removed